lib/puppet/provider/bootstrap_rpm/bootstrap_rpm.rb

```diff
@@ -100,6 +100,8 @@ def build_rpm
 
   def copy_rpm
     FileUtils.copy(built_rpm, resource[:dest])
+    File.chmod(resource[:mode].to_i(8), latest_rpm)
+    FileUtils.chown(resource[:owner], resource[:group], latest_rpm)
```

Comment:
```suggestion
    latest = latest_rpm
    File.chmod(resource[:mode].to_i(8), latest)
    FileUtils.chown(resource[:owner], resource[:group], latest)
```

avoids repeating some (relatively) expensive operations in latest_rpm